blog/content/post/2019-06-21-rte-new-version-and-kit.md

```diff
@@ -0,0 +1,128 @@
+---
+title: RTE v1.1.0 - enhanced remote testing now available!
+abstract: Get familiar with new RTE revision - what has been changed, which
+          features are now more complex and which items has been added to the
```

Comment:
```suggestion
          features are now more sophisticated and which items have been added to the
```

@artur-rs I'm not sure what you mean by `complex`, but let's not scare potential buyer :)